tests/isomorphism_problem.py

```diff
@@ -105,11 +105,11 @@
             print('---------------------------')
 
             start_isomorph = time()
-            boolean_isomorph = are_isomorph(G, H)
+            boolean_isomorph = are_isomorph(G.copy(), H.copy())
             end_isomorph = time()
 
             start_amount_isomorphisms = time()
-            n_isomorphisms = amount_of_isomorphisms(G, H)
+            n_isomorphisms = amount_of_isomorphisms(G.copy(), H.copy())
```

Comment:
Je gebruikt G en H alleen op deze twee veranderde regels, is het dan niet genoeg om alleen op de eerste de copy te doen?

Reply:
Klopt, hier kan ik ze gewoon in stoppen, want hierna wordt toch niks meer getest.